Google.Api.Gax.Grpc/CallSettings.cs

```diff
@@ -47,6 +68,38 @@ public sealed class CallSettings
         /// Constructs an instance with the specified settings.
         /// </summary>
         /// <param name="cancellationToken">Cancellation token that can be used for cancelling the call.</param>
+        /// <param name="expiration"><see cref="Expiration"/> to use, or null for default expiration behavior.</param>
+        /// <param name="retry"><see cref="Retry"/> to use, or null for default retry behavior.</param>
+        /// <param name="headerMutation">Action to modify the headers to send at the beginning of the call.</param>
+        /// <param name="writeOptions"><see cref="global::Grpc.Core.WriteOptions"/> that will be used for the call.</param>
+        /// <param name="propagationToken"><see cref="ContextPropagationToken"/> for propagating settings from a parent call.</param>
+        /// <param name="responseMetadataHandler">Action to invoke when response metadata is received.</param>
+        /// <param name="trailingMetadataHandler">Action to invoke when trailing metadata is received.</param>
+        public CallSettings(
+            CancellationToken? cancellationToken,
+            Expiration expiration,
+            RetrySettings retry,
+            Action<Metadata> headerMutation,
+            WriteOptions writeOptions,
+            ContextPropagationToken propagationToken,
+            Action<Metadata> responseMetadataHandler,
+            Action<Metadata> trailingMetadataHandler)
+        {
```

Comment:
Can this just delegate to another constructor? (I think it's fine to delegate to an obsolete one.)

Reply:
Yes, absolutely - that's what I'd expected.